### PR TITLE
[Content] Getting started with Abstract tutorial

### PR DESCRIPTION
**Pull request by ronijaakkola**
## Description
This PR adds a tutorial page to the documentation site about how to get started with using Abstract. It goes through the basic project setup process and also how the designer can make their first branch and design changes.

## Related Issue
https://helsinkisolutionoffice.atlassian.net/browse/HDS-573

## Motivation and Context
HDS documentation was lacking a guide for basic Abstract setup and use. Many designers in Helsinki have not used Abstract before so this will help them to get started.

## How Has This Been Tested?
Tested by running the documentation site locally.

### CODE REVIEW

**Review by laurakarhu — APPROVED** [review]
Looking good, thanks!

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+<p>Abstract has both a desktop and a web app. If you are just viewing design files, you can use the web app. If you are a designer, you need to download the desktop app to edit Sketch files. You can download the app from the <Link href="https://app.abstract.com/download" external>Abstract website download page</Link>. Note that you will need a Mac computer to use the Abstract desktop app.</p>
+
+### Which Abstract role do you need?
+Abstract has two (2) levels of access rights; **Viewer** and **Contributor**. 
+
+Users with the **Viewer** role can browse projects and their files and view detailed design specifications in the Inspect mode. Users with the **Contributor** role have the same features as the Viewer role does but also can create and edit Abstract projects. 
+
+As a rule of thumb, if you need to create or edit design files in Abstract, you need the Contributor role.
+
+<p>If you already are part of the City of Helsinki organisation but missing the Contributor role, ask the city of Helsinki representative of your project to fill out the <Link href="https://forms.office.com/r/ebx0YFYspB" external>Abstract invite form</Link> for you (requires Helsinki AD credentials).</p>
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:34]: If you already are part of the City of Helsinki organization but missing the Contributor role, ask the city of Helsinki representative of your project to fill out the

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+### How to get invited to the City of Helsinki Abstract organisation?
+All City of Helsinki Abstract projects are under an organisation. To access these projects, you need an invite.
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:37]: Every City of Helsinki Abstract project is under an organization.

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+<p>First, you need to register to Abstract. You can do that on the <Link href="https://app.abstract.com/signup" external>Abstract sign up page</Link>. You may use any email address you like, for example the email of your own organisation.</p>
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:39]: You may use any email address you like, for example, the email of your organization.

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+<p>After you have your account, you can get invited to the organisation. Ask the city of Helsinki representative of your project to fill out the <Link href="https://forms.office.com/r/ebx0YFYspB" external>Abstract invite form</Link> for you (requires Helsinki AD credentials).</p>
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:41]: After you have your account, you can get invited to the organization

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+<Image src="../../static/tutorials/abstract-getting-started/abstract-new-project-button.png" alt="Abstract create new project button" style="width: 100%; max-width:850px; margin-bottom:24px;" viewable/>
+
+If you cannot see the "New Project" button, you are most likely missing the Contributor role. See the [above question "Which Abstract role do you need?"](#which-abstract-role-do-you-need) for more information.
+
+<Image src="../../static/tutorials/abstract-getting-started/abstract-create-new-project.png" alt="Adding a project description in Abstract" style="width: 100%; max-width:850px; margin-bottom:24px;" viewable/>
+
+Next, input a descriptive name for your project. It is recommended not to select "Make project private" unless you have a specific reason to do so. This way your project will public for all City of Helsinki designers.
+
+Lastly, open your newly created project and write a description for it by selecting "Edit" at the top-right corner of the "About" area. It is recommended to briefly describe the goal of the project and also provide contact information of the people responsible of the design files.
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:56]: contact information of the people responsible for the design files.

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+If you want to create a new Sketch file, choose the "Create new file..." option. If you want to import an existing Sketch file, select "Import File..." option.
+
+### How to link HDS libraries to a project?
+To be able to use HDS components in your design files, you need to link the design libraries to the project. To link the HDS libraries, open the project in the Abstract desktop app. Browse to the "Master" page and the "Files" tab. Select the "Add File" button and under it select the option "Link Library...".
+
+<Image src="../../static/tutorials/abstract-getting-started/abstract-link-libraries-button.png" alt="Link libraries button in Abstract" style="width: 100%; max-width:850px; margin-bottom:24px;" viewable/>
+
+On the left panel, find and select the project "Helsinki Design System". After this, select all files that appear on the right panel. You can select them quickly by selecting the first one, scrolling to the bottom of the list and select the last one with the Shift key held down. Last, press the "Link Library" to finish the process.
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:75]: ottom of the list, and select the last one with the Shift key held down

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+When you have made all changes to the branch, it is time to **Merge** it to the Master Branch. Merging means taking all of your new design changes to combining them with the designs in the Master Branch - creating a new source of truth for your project. To start the merge, press "Merge branch..." (or "Update From Master") button at the top-right corner of the branch screen. You may skip the merge description and press "Merge and Archive" button to continue.
+
+<Image src="../../static/tutorials/abstract-getting-started/abstract-merge-branch-button.png" alt="Merge branch button in Abstract" style="width: 100%; max-width:850px; margin-bottom:24px;" viewable/>
+
+Next, Abstract checks if there are any **Conflicts** in your merge. Conflicts mean that someone else (another designer) has made changes to the Master branch that conflict with the changes you just made. Abstract provides easy-to-use interface to select whether you want to use your changes or the changes in the Master Branch. Make the selection for each artboard/symbol to finalise the merge.
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:116]: Abstract provides an easy-to-use interface to select whether you want to use your changes or the changes in the Master Branch. Make the selection for each artboard/symbol to finalize the merge.

File: site/docs/tutorials/abstract-getting-started.mdx [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+<Image src="../../static/tutorials/abstract-getting-started/abstract-merge-conflicts.png" alt="Resolving merge conflicts in Abstract" style="width: 100%; max-width:850px; margin-bottom:24px;" viewable/>
+
+Congratulations, you have just made your first design change and committed it to the Master Branch!
+
+## Abstract resources
+Abstract has a plenty of features that were not covered in this short guide. Below resources help you to get started with more advanced features. You may also contact the HDS team for guidance if needed. See [HDS Support page](/about/support) for more information.
```
Comment (minevala) [inline site/docs/tutorials/abstract-getting-started.mdx:123]: Abstract has plenty of features that were not covered in this short guide. The Below resources help you to get started with more advanced features

**Review by minevala — COMMENTED** [review]
Grammar check 😁

File: site/docs/get_started/for_designers.mdx [hunk 9]
```diff
@@ -52,83 +52,31 @@ If you are a newcomer to Sketch or Abstract, they both offer some great tutorial
...
-
-
-2. From the _Components_ tab in the sidebar on the left, by selecting _HDS [component name]_. After selecting a HDS component, you’ll see the variations and states of that component. Select the desired variant from the sidebar, and drag it to your artboard. 
-
-You can also select _All_ and filter the components list by component or variant name.
-
-<Image src="../../static/fordesigners-component-insert.png" alt="Inserting component from Components tab" style="width: 100%; max-width:850px" />
+**If you are new to Abstract**, please read [HDS Getting started with Abstract tutorial](/tutorials/abstract-getting-started) for more information on how to set up and use Abstract.
 
+## Customising HDS components
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply site/docs/get_started/for_designers.mdx:65]: HDS uses British English where "customi**s**e" is the correct form.

File: site/docs/get_started/for_designers.mdx [hunk 10]
```diff
@@ -52,83 +52,31 @@ If you are a newcomer to Sketch or Abstract, they both offer some great tutorial
...
+HDS components are made to be customisable to allow expression of the vibrant Helsinki brand.
```
[the remark this answers is not in the document]
Comment (ronijaakkola) [reply site/docs/get_started/for_designers.mdx:66]: HDS uses British English where "customi**s**able" is the correct form.

**Review by minevala — APPROVED** [review]

**Pull request merged**